README.rst

```diff
@@ -92,10 +92,14 @@ The minimum required dependencies to run MNE-Python are:
 - Python >= 3.7
 - NumPy >= 1.18.1
 - SciPy >= 1.4.1
+- Matplotlib >= 3.1.0
+- pooch >= 1.5
+- tqdm
+- Jinja2
+- decorator
```

Comment:
I guess we do not need `packaging` because some of these packages depends on it? Or should we add it once we're using it directly (#10197)?

Reply:
We can add it in #10197